fortran/test/unit/test_musica_fortran_interface.F90

```diff
@@ -1,28 +1,42 @@
 program test_micm_fort_api
   use iso_c_binding
-  use micm_core, only: micm_t
+  use micm_core, only: micm_t, mapping_t
 
   implicit none
 
   type(micm_t), pointer         :: micm
   real(c_double)                :: time_step
   real(c_double)                :: temperature
   real(c_double)                :: pressure
-  integer(c_int)                :: num_concentrations
-  real(c_double), dimension(5)  :: concentrations
-  integer                       :: errcode
-  character(len=7)              :: config_path
+  integer(c_int)                :: num_concentrations, num_user_defined_reaction_rates
+  real(c_double), dimension(5)  :: concentrations 
+  real(c_double), dimension(3)  :: user_defined_reaction_rates 
+  integer                       :: errcode, i
+  character(len=256)            :: config_path
+  type(mapping_t)                 :: the_mapping
 
   time_step = 200
   temperature = 272.5
-  pressure = 101253.3
+  pressure = 101253.4
```

Comment:
haha just curious, what made you change the decimal number?

Reply:
I don't have a memory of even doing that